src/test/java/seedu/taassist/logic/commands/CommandTestUtil.java

```diff
@@ -34,8 +34,8 @@ public class CommandTestUtil {
     public static final String VALID_EMAIL_BOB = "bob@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_CLASS_HUSBAND = "husband";
```

Comment:
maybe we can change the `HUSBAND` and `FRIEND`(in the previous line) later too! Putting this comment here in case we forget in the future